laba 5 header

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <math.h>
+
+using namespace std;
+
+class RaceHorse {
+private:
+    string name;
+    int age;
+    double speed;
+    int place_in_race;
+public:
+    string get_name() const {
+        return name;
+    }
+    int get_age() const {
+        return age;
+    }
+    double get_speed() const {
+        return speed;
+    }
+    void set_place(int place) {
+        place_in_race = place;
+    }
+    double get_place() const {
+        return place_in_race;
+    }
+    RaceHorse(string name, double speed, int age) :name(name), speed(speed), age(age) {};
+};
+
+void print(RaceHorse horse) {
```

Comment:
Function must start with capital letter.
What exactly does your function print? The name should be more specific